meta-mender-core/recipes-mender/mender-connect/mender-connect.inc

```diff
@@ -13,19 +13,13 @@ SYSTEMD_SERVICE:${PN} = "mender-connect.service"
 
 B = "${WORKDIR}/build"
 
-inherit go
+inherit go-mod
```

Comment:
Note to self,

This change is the culprit of the CI failures. Changing the compile steps has added two dependencies on `mender-connect`:
* go-runtime >= 1.17.13
* libstd.so()(64bit)

Which increased the size of the final image making it not fit for one of the configurations.

To investigate:
* Why?
* Does it affect the same way `scarthgap` builds?
* Should we revert or just increase the image size?

Reply:
See:
* https://github.com/mendersoftware/meta-mender/pull/2221